CanalTP/Chaos
# Update swagger 2 to openapi 3

# Description

This PR update the swagger documentation to use openapi 3

Note:
- it should help also to give a better example of a PUT Disruption with an Impact ID
- a better description of a ptObject type line-section (specific meta object)

## Issue

Issue link: BOT-1680

## How to test

Here are the following steps to test this pull request:

check with this address : 
http://petstore.swagger.io/?url=https://raw.githubusercontent.com/CanalTP/Chaos/task-bot-1680-openapi-3/documentation/swagger.yml
